src/main/java/seedu/address/model/person/Information.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Customer's information in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidInformation(String)}.
+ */
+public class Information {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Information should not be blank, and it can contain any characters.";
+
+    /*
+     * The information field should not be blank.
+     */
+    public static final String VALIDATION_REGEX = "^(?!\\s*$).+";
+
+    public final String value;
+
+    /**
+     * Constructs an {@code Information}.
+     *
+     * @param information A valid information string.
+     */
+    public Information(String information) {
+        requireNonNull(information);
+        checkArgument(isValidInformation(information), MESSAGE_CONSTRAINTS);
```

Comment:
It is better to streamline customer's order history and favorite items within customer's information